crates/api/src/lib.rs

```diff
@@ -141,6 +150,52 @@ pub(crate) fn build_totp_2fa(
   .with_lemmy_type(LemmyErrorType::CouldntGenerateTotp)
 }
 
+/// Since removals and bans are only federated for local users,
+/// you also need to send bans for their content to local communities.
+/// See https://github.com/LemmyNet/lemmy/issues/4118
+#[tracing::instrument(skip_all)]
+pub(crate) async fn send_bans_and_removals_to_local_communities(
```

Comment:
How about `ban_user_in_local_communities()`? The comment is also not so clear about what the function actually does.

Reply:
I renamed to something better.